src/main/java/mate/academy/onlinebookstore/dto/shoppingcart/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.onlinebookstore.dto.shoppingcart;
+
+import jakarta.validation.constraints.Digits;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+
+public record CreateCartItemRequestDto(
+        @NotNull
+        @Positive
+        @Digits(integer = 19, fraction = 0)
+        Long bookId,
+        @NotNull
```

Comment:
```suggestion
```

Reply:
I don't quite understand why @Notnull is not needed here?